docs/_includes/ref-packtoc.html

```diff
@@ -2,8 +2,14 @@
 {%- for package in site.ref_packages %}
 
 <li>
-<strong>{{package | escape}}</strong>
```

Comment:
I'm guessing all the changes to this file are experimental/scratch changes. Should they be removed?